catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -384,6 +384,18 @@
             type="Boolean"
             default="false"
             required="false"/>
+
+        <AD name="Upload Editor: Attribute Configuration" id="attributeEnumMap" required="false" type="String" cardinality="100"
+            default=""
```

Comment:
❓ Do we want defaults here so that the ui portion isnt blank by default?

Reply:
oh i missed that it goes away completely if nothing's configured. That makes sense - ignore my comment 😅